app-text/pandoc/pandoc-3.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Metapackage for pandoc version 3"
+HOMEPAGE="https://pandoc.org/"
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+
+RDEPEND="app-text/pandoc-cli"
```

Comment:
Should there be an `elog` message in `pkg_postinst` letting the users know that this is now a metapackage and that the real package is `app-text/pandoc-cli`?

Reply:
Any suggestion for a wording?

Reply:
```ebuild
elog "The pandoc CLI executable has been split off upstream into a new package named"
elog "pandoc-cli, starting with pandoc-3. This metapackage ${P}"
elog "was created to help ease upgrading, but it is recommended to switch to"
elog "app-text/pandoc-cli."
```